### PR TITLE
[sphinx] Fix a few warnings.

### PR DESCRIPTION
**Pull request by Zimmi48**

### CODE REVIEW

File: doc/sphinx/addendum/miscellaneous-extensions.rst [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
 
 Miscellaneous extensions
-=======================
+========================
 
+.. what is this about?
 .. contents::
    :local:
    :depth: 1
-----
```
Comment (Zimmi48) [inline doc/sphinx/addendum/miscellaneous-extensions.rst]: This was emitting an "unexpected unindent warning". However, I'd like to understand what this block is supposed to do.

File: doc/sphinx/language/cic.rst [hunk 2]
```diff
@@ -1278,7 +1278,7 @@ and :math:`I:A` and :math:`λ a x . P : B` then by :math:`[I:A|B]` we mean that
 :math:`λ a x . P` with :math:`m` in the above match-construct.
 
 
-.. _Notations:
+.. _cic_notations:
```
Comment (Zimmi48) [inline doc/sphinx/language/cic.rst]: This label was defined twice.

**Comment by Zimmi48** [comment]
Is there a better way to see all the warnings again than to delete the `doc/sphinx/_build` directory? I didn't find any `sphinx-clean` target.

File: doc/sphinx/addendum/miscellaneous-extensions.rst (hunk 1 above)
Comment (maximedenes) [reply doc/sphinx/addendum/miscellaneous-extensions.rst]: This block should have been removed.

**Comment by Zimmi48** [comment]
I took the number of warnings down from 300 to just 50.

**Comment by maximedenes** [comment]
Superseded by and contained in #7239.

**Pull request closed without merging**